src/function/open-referral-function/src/FamilyHubs.OpenReferral.Function/Repository/FunctionDbContext.cs

```diff
@@ -0,0 +1,24 @@
+using FamilyHubs.ServiceDirectory.Data.Entities.Staging;
+using Microsoft.EntityFrameworkCore;
+
+namespace FamilyHubs.OpenReferral.Function.Repository;
+
+public class FunctionDbContext(DbContextOptions<FunctionDbContext> options) : DbContext(options), IFunctionDbContext
+{
+    public DbSet<ServicesTemp> ServicesTemp { get; init; } = null!;
+
+    public void AddServiceTemp(ServicesTemp serviceTemp) => ServicesTemp.Add(serviceTemp);
+
+    public Task TruncateServicesTempAsync() => Database.ExecuteSqlRawAsync("TRUNCATE TABLE [staging].[services_temp]");
+
+    public Task<int> SaveChangesAsync() => base.SaveChangesAsync();
+
+    protected override void OnModelCreating(ModelBuilder modelBuilder)
+    {
+        modelBuilder.Entity<ServicesTemp>()
+            .ToTable("services_temp", "staging")
```

Comment:
nitpick: Since we're moving away from using "staging" as our domain language to describe this stuff to "deds", should the schema be changed to `deds`?

Reply:
Good point, I could raise another PR to change this in the SD API as well -- saying that we are getting rid of the services_temp Db next so it might be worth keeping this for now and then getting rid of it as part of those tickets? Don't mind doing either way :)